builds/e2e/templates/e2e-setup.yaml

```diff
@@ -179,6 +210,7 @@ steps:
         loadGenImage = "$imagePrefix-load-gen:$imageTag";
         relayerImage = "$imagePrefix-relayer:$imageTag";
         networkControllerImage = "$imagePrefix-network-controller:$imageTag";
+        testBlobStoreSas = $TestBlobStoreSas;
```

Comment:
Does this work? I'd expect that you either need to surround the pipeline var in parens, e.g., `$(TestBlobStoreSas)`, or you have to use the bash variable, which should be ALL CAPS, e.g., `$TESTBLOBSTORESAS`.

Reply:
Well, the tests pass and I verified logs were uploaded.